showcase/src/app/getting-started/getting-started-component/getting-started.component.html

```diff
@@ -0,0 +1,217 @@
+<div class="getting-started-content">
+	<div class="mat-display-2">Getting started</div>
+
+	<table-of-contents headerSelectors=".custom-title,.custom-subtitle" class="toc-list"></table-of-contents>
```

Comment:
Could you please try to use the **container** attribute to test if it is working correctly ? 😊

Reply:
I've actually removed it. We (@christophercr and I) have tried several times to make it work, but this is particular to angular material way of implementing pages. It was never displayed when we put the container. I think it is best to stick to the page in our case.